src/tab/header/item/TabHeaderItem.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+import {TabItem} from "../../Tab";
+import Button from "../../../button/Button";
+
+type TabHeaderItemProps = Omit<TabItem, "id"> & {
+  index: number;
+  onClick: (index: number) => void;
+};
+
+function TabHeaderItem({children, icon, index, onClick}: TabHeaderItemProps) {
+  return (
+    <Button
+      customClassName={"tab-header-item"}
+      onClick={handleClick}
+      testid={`tab-header-item-${index}`}>
```

Comment:
Let's pass `testid` as props to this component. Similarly, I think `Tab` component should also get a `testid` and the testid for each TabHeaderItem should be `${testidForTab}.TabHeaderItem-${index}`

Reply:
Filed an issue https://github.com/Hipo/react-ui-toolkit/issues/12